### ISSUE TITLE
Activity are not showing disarmed events

### ISSUE THREAD

**Issue by AndersNorlev** [akasma74/Hass-Custom-Alarm#110]
**The component:**
 - Current version: v1.12.6
 - Last known working version (if any):

**Your setup:**
 - Home Assistant version: 0.112.4
 - Last known working Home Assistant version (if any):
 - OS: Windows 10
 - Browser name & version: Chrome 80.0.3987.163

**Describe the bug**
In alarm panel under Activity, its only showing when the alarm is set the alarm in Away mode.

**Comment by AndersNorlev** [comment]
After more testning i can see that trigged also is showing in activity.

**Comment by akasma74** [comment]
I don't understand what it the matter of your issue, could you elaborate (perhaps using screenshot and few more words to explain)?

**Comment by AndersNorlev** [comment]
![Screenshot_20200723-230127.jpg](https://user-images.githubusercontent.com/60581646/88338321-7a2f3c00-cd38-11ea-928a-5dca3eda54c9.jpg)
I Will try 😊
In the activity log, i cant see when the alarm is disarmed.
I have just tried to armed home and disarmed, but only armed home is in the log.

**Comment by akasma74** [comment]
So basically you can see anything apart from `alarm disarmed`, is that correct?

**Comment by AndersNorlev** [comment]
Yes 😊

**Comment by akasma74** [comment]
Well.. mine works as expected
![Screen Shot 2020-07-23 at 22 16 18](https://user-images.githubusercontent.com/13675855/88339633-4f41e980-cd32-11ea-9dbd-3d75feacf4e4.png)

Could you post your `bwalarm.yaml` here?

**Comment by AndersNorlev** [comment]
Yes, its comming here :-)
platform: bwalarm
panel:
  cameras: []
  panel_title: ''
  camera_update_interval: ''
  enable_fahrenheit: 'False'
  enable_sensors_panel: 'True'
  hide_passcode: 'True'
  hide_sidebar: 'True'
  enable_clock: 'False'
  enable_floorplan_panel: false
  enable_camera_panel: 'False'
  enable_custom_panel: false
  round_buttons: 'True'
  shadow_effect: 'False'
enable_night_mode: false
states:
  armed_away:
    immediate:
    - binary_sensor.gang_sensor_motion
    delayed: []
    override: []
    pending_time: 0
    warning_time: 0
    trigger_time: 600
  armed_home:
    immediate: []
    delayed: []
    override: []
    pending_time: 0
    warning_time: 0
    trigger_time: 600
  armed_night:
    immediate: []
    delayed: []
    override: []
    pending_time: 0
    warning_time: 0
    trigger_time: 600
users:
- id: !ID
  name: anders
  enabled: true
  code: '!Pin'
  picture: ha.png
admin_password: !Password
code_to_arm: true
enable_log: true
log_size: 20

**Comment by akasma74** [comment]
Thanks.
Could you edit your last post to preserve formatting?
It can be done by enclosing the config content within three back quotes (or selecting the text and hitting `Insert code` above the message editor)

**Comment by AndersNorlev** [comment]
Nice feature 😊

**Comment by akasma74** [comment]
that's better.
one more thing - could you scramble your sensitive details like passwords etc rather than delete them?
the thing is it's YAML and a lot of things have impact on the resulting values so I want as close copy of your config as possible to reproduce the issue.

**Comment by AndersNorlev** [comment]
Of cause 😊
At the phone so thats was faster

**Comment by akasma74** [comment]
@AndersNorlev I'm still waiting for the scrambled config

**Comment by AndersNorlev** [comment]
Oh a copy of My configtion.yaml?

**Comment by akasma74** [comment]
No, just update [this post](https://github.com/akasma74/Hass-Custom-Alarm/issues/110#issuecomment-663342474) with content of your `bwalarm.yaml` but instead of deleting sensitive info like passwords etc change them to 1234 or so.

By the way, you can use `secrets.yaml` so instead of real values you can use `!secret alarm_admin_password` in your `bwalarm.yaml` - thay way you can easily share it with others without the need to hide that sensitive info every time ;)

**Comment by AndersNorlev** [comment]
Its should be updated now :-)

okay nice, i will try to move it to secrets :-)

**Comment by akasma74** [comment]
thanks

**Comment by AndersNorlev** [comment]
I can see in the logbook that the alarm is disarmed 
![image](https://user-images.githubusercontent.com/60581646/89189599-7d2ff500-d5a0-11ea-95b5-a2c208f5732f.png)

Do you know how to create a notification with who as armed or disarmed the alarm?

**Comment by akasma74** [comment]
> I can see in the logbook that the alarm is disarmed

that's as expected. otherwise it won't disarm ;)

> Do you know how to create a notification with who as armed or disarmed the alarm?

I don't think it's possible outside the component as information about users is in its config.
Do you really need it?
